lib_i2c/src/i2c_master.xc

```diff
@@ -68,7 +69,8 @@ static void release_clock_and_wait(
   // executing then the clock needs to be adjusted
   const int wake_up_ticks = 10;
   if (time > fall_time + delay + wake_up_ticks) {
```

Comment:
Isn't there an issue if time wraps here? Should it not use timeafter() from xs1.h?